shared-libs/transitions/src/lib/schedules.js

```diff
@@ -10,6 +10,16 @@ const messages = require('../lib/messages');
 const messageUtils = require('@medic/message-utils');
 const mutingUtils = require('../lib/muting_utils');
 
+const isMuted = (patient, place) => {
+  if (patient) {
+    return patient.muted || mutingUtils.isMutedInLineage(patient);
+  }
+
+  if (place) {
+    return place.muted || mutingUtils.isMutedInLineage(place);
+  }
+};
```

Comment:
> const muted = isMuted(patient) || isMuted(place);

This behaves differently from the current implementation. 
The current implementation prioritizes evaluating the patient's muted state. If the patient is unmuted, we don't even check the place. This is intentional. 
With the suggested implementation, if either the patient or the place are muted, the schedules are muted. 

I chose this implementation to make the intention of prioritizing the patient more obvious, by not treating patient and place equally. 

Do you think we should treat patient and place equally?

Reply:
As an addendum to the above comment, the patient and the place don't have to be part of the same lineage. So it can be a perfectly valid situation where the patient is unmuted but the place is.

Reply:
For example, the message is "Jesse is visiting her aunt, Petunia.". The schedule is "Ask Jesse if she enjoyed the trip.". Jesse is unmuted, but Petunia is muted. Should the followup be scheduled or muted in this case?

Reply:
I updated the code and added a comment. Please let me know what you think.

Reply:
I'm not sure whether it makes sense to have a muted place with an unmuted patient, but let's keep the implementation consistent with what was there before. I like the new code, thanks!